.github/workflows/chatbot.yaml

```diff
@@ -14,12 +14,16 @@ on:
     #   - ./recipes/natural_language_processing/chatbot/**
     #   - .github/workflows/chatbot.yaml
 
+permissions: write-all
+
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository_owner }}/chatbot
+  TEST_IMAGE_NAME: ${{ github.repository_owner }}/chatbot
+  QUAY: quay.io
```

Comment:
oh ok! did you already add that? this is the only reason I'm adding anything here :) - let's see what's there, perhaps this PR can go away then!

Reply:
I'd like to start pushing the chatbot & llamacpp-python image in quay.io/ai-lab now with merges - since I'm currently manually building & pushing from local

Reply:
@lmilbaum I see what needs to happen here, I'll update this :)

Reply:
How about adding a step to the testing-framework workflow to promote the images to quay.io?